app/src/androidTest/java/ch/hikemate/app/ui/map/MapScreenTest.kt

```diff
@@ -138,11 +157,29 @@ class MapScreenTest : TestCase() {
 
   @Test
   fun emptyHikesListDisplaysEmptyMessage() {
+    setUpMap()
     `when`(hikesRepository.getRoutes(any(), any(), any())).thenAnswer {
       val onSuccess = it.getArgument<(List<HikeRoute>) -> Unit>(1)
       onSuccess(listOf())
     }
     composeTestRule.onNodeWithTag(MapScreen.TEST_TAG_SEARCH_BUTTON).performClick()
     composeTestRule.onNodeWithTag(MapScreen.TEST_TAG_EMPTY_HIKES_LIST_MESSAGE).assertIsDisplayed()
   }
+
+  /**
+   * Regression test for the app crashing when searching for hikes while zoomed out too far.
+   *
+   * See https://github.com/HikeMate/hikeMateApp/issues/78 for more details.
+   */
+  @Test
+  fun zoomedOutMapDoesNotCrashWhenSearchingForHikes() {
+    // Initialize the map as zoomed out very far
+    setUpMap(mapMinZoomLevel = 1.0, mapInitialZoomLevel = 1.0)
```

Comment:
Question: Just to clarify, you have a `setUpMap()` on every test because you need to alter the parameters here?

Reply:
Yes, initially I could do that in the `setUp` function, but I need to change the zoom for this unique test so that it actually tests what is expected.

As a consequence, I had to move all the `setContent` calls to the tests themselves, otherwise I get an error `Error : cannot call setContent twice in a test`.